test/src/waittask.spec.ts

```diff
@@ -609,7 +699,7 @@ describe('waittask specs', function () {
     it('should have an error message specifically for awaiting an element to be hidden', async () => {
       const {page} = getTestState();
 
-      await page.setContent(`<div></div>`);
```

Comment:
@OrKoN It seems tests weren't covering intended behavior, just programmed behavior ;)

Reply:
I actually wonder if it's going to be a breaking change since people likely rely on this? Anyway I think we should fix it but perhaps see if we mark it as a breaking change and land after all the refactoring and features.

Reply:
Could we land it together? All the refactors and features are on top of this.

Reply:
Let's mark it as non breaking then as it's technically a fix. Let's include the definition of what a visible bounding box is in the commit description.

Reply:
I've marked it as a breaking fix. It is still breaking, but it's a fix :P